trieste/utils/pareto.py

```diff
@@ -148,3 +149,44 @@ def hypervolume_indicator(self, reference: TensorType) -> TensorType:
         hypervolume = tf.reduce_sum(tf.reduce_prod(upper - lower, 0))
 
         return tf.reduce_prod(reference[None] - min_pfront) - hypervolume
+
```

Comment:
iirc, you'll need to make the bounds functionality private, which is going to mean you'll have to rewrite a fair few tests

Reply:
I am not quite following here, do you mean the `pareto.bounds`?  If this is the case, from application perspective, it is not so far needed for acquisition function

Reply:
yes, making `.bounds` private

Reply:
Made private, the test is not modified too much as there is only one test verifying `_bounds` numerical correctness, so I was hoping this wouldn't be a big issue here, CMIIW.